src/app/+run-tale/run-tale/tale-versions-panel/tale-versions-panel.component.html

```diff
@@ -0,0 +1,64 @@
+<div id="timeline-column">
+  <div class="ui vertical segment">
+    <div style="margin-bottom: 1rem">
+        <button class="ui button fluid" (click)="saveNewVersion()">
+        <i class="fas fa-save icon"></i>Save Tale Version
+        </button>
+    </div>
+
+    <!-- TODO: Recorded Runs configurations -->
+    <div class="ui fluid selection version dropdown" tabindex="0" *ngIf="false">
+        <input type="hidden" name="Configuration">
+        <i class="dropdown icon"></i>
+        <div class="default text">Select recorded run configuration...</div>
+        <div class="menu" tabindex="-1">
+          <div class="item" data-value="1">Default</div>
+          <div class="item" data-value="0">My Custom Config</div>
+          <div class="divider"></div>
+          <div style="margin: .75rem .75rem .35rem">
+            <button class="ui button fluid" (click)="editRunConfigurations()">
+              Edit run configurations
+            </button>
+          </div>
+        </div>
+    </div>
+
+    <!-- TODO: Recorded Runs button -->
+    <div style="margin-top: .25rem" *ngIf="false">
+        <button class="ui button fluid primary" (click)="performRecordedRun()">
+          <i class="fas fa-check-circle icon"></i>
+          Recorded Run
+        </button>
+    </div>
+  </div>
+  <h3>Tale History</h3>
+  <div id="timeline" class="ui middle aligned divided list">
+    <div class="ui placeholder" style="height:100%;" *ngIf="!timeline.length">
```

Comment:
May I suggest: 
```suggestion
    <div class="ui message" style="height:100%;" *ngIf="!timeline.length">
```
in the spirit of https://github.com/whole-tale/ngx-dashboard/pull/114 ?

Reply:
Committed this externally... I don't trust Github UI and there were other conflicts to fix